privaterelay/tests/mgmt_update_phone_remaining_stats_tests.py

```diff
@@ -304,3 +304,15 @@ def test_phone_subscriber_with_subscription_end_date_after_reset_phone_limits_up
     assert num_profiles_updated == 1
     relay_number.refresh_from_db()
     assert relay_number.remaining_texts == settings.MAX_TEXTS_PER_BILLING_CYCLE
+
+
+@pytest.mark.django_db
+def test_update_phones_command(capsys):
+    call_command("update_phone_remaining_stats")
+    out, err = capsys.readouterr()
```

Comment:
(comment) I learned [something new about pytest](https://docs.pytest.org/en/6.2.x/capture.html#accessing-captured-output-from-a-test-function) today about capturing stdout and err.